packages/oui-navbar/src/navbar.html

```diff
@@ -74,21 +74,21 @@
             ng-attr-aria-label="{{::!!asideLink.label ? asideLink.label : null}}"
             ng-attr-title="{{::!!asideLink.label ? asideLink.label : null}}"
             ng-class="::!!asideLink.iconClass ? 'oui-navbar-link_icon' : null"
-            ng-click="$ctrl.toggleMenu(asideLink.name)"
+            ng-click="$ctrl.toggleMenu(asideLink.name); asideLink.onClick();"
```

Comment:
No check for `onClick()` if undefined ? 

What do you think about adding a parameter for the toggleMenu method `toggleMenu (state, isInternalNav, callback)` and check if is function ?

Reply:
Is it a bit weird to call ```$ctrl.toggleMenu(asideLink.name, false, asideLink.onClick())```
```toggleMenu``` is already used at multiple place in navbar.html

Reply:
In fact, I had more in mind
```
$ctrl.toggleMenu(asideLink.name, false, asideLink.onClick)
```
and in the method
```
if (angular.isFunction(callback)) {
   callback();
}
```
This way, it will not disturb the others

Reply:
Le problème est que ```asideLink.onClick``` prend en argument ```asideLink``` qui n'est pas visible dans le controlleur